### PR TITLE
Updating create-package to work with storybook, typed css modules

### PR DESCRIPTION
**Pull request by dzearing**
Addresses issue: https://github.com/microsoft/fluentui/issues/12365

Cleaning up a variety of things in `yarn create-package`:

- [x] `just.config.js` should be `just.config.ts`
- [x] scope for new packages should be `@fluentui`
- [x] remove legacy demo app config
- [x] add storybook config
- [x] look at all auto-generated files to ensure they're relevant
- [x] change Fabric references to Fluent
- [x] change Office 365 references to Microsoft 365 or remove entirely

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/fluentui/pull/12492)

### CODE REVIEW

File: scripts/templates/create-package/EmptyPackageJson.mustache [hunk 1]
```diff
@@ -33,21 +36,19 @@
...
     "es6-weak-map": "{{{es6WeakMap}}}",
     "react": "{{{react}}}",
     "react-app-polyfill": "{{{reactAppPolyfill}}}",
     "react-dom": "{{{reactDom}}}",
-    "react-test-renderer": "{{{reactTestRenderer}}}"
+    "react-test-renderer": "{{{reactTestRenderer}}}",
+    "typescript-plugin-css-modules": "{{{typescriptPluginCssModules}}}"
+
   },
   "dependencies": {
```
Comment (ecraig12345) [inline scripts/templates/create-package/EmptyPackageJson.mustache:50]: IIRC load-themed-styles is needed to make sass work?

File: scripts/templates/create-package/EmptyPackageJson.mustache [hunk 2]
```diff
@@ -33,21 +36,19 @@
...
-    "@microsoft/load-themed-styles": "{{{microsoftLoadThemedStyles}}}",
     "@uifabric/set-version": "{{{uifabricSetVersion}}}",
```
Comment (ecraig12345) [inline scripts/templates/create-package/EmptyPackageJson.mustache:51]: Do we need this?

File: scripts/create-package.js [hunk 3]
```diff
@@ -51,20 +51,17 @@ const steps = [
...
   { template: 'Readme', output: 'README.md' },
-  { template: 'JustConfig', output: 'just.config.js' },
+  { template: 'JustConfig', output: 'just.config.ts' },
   { template: 'JestConfig', output: 'jest.config.js' },
   { template: 'PackageJson', output: 'package.json' },
   { template: 'TsConfig', output: 'tsconfig.json' },
   { template: 'TsLint', output: 'tslint.json' },
-  { template: 'WebpackConfig', output: 'webpack.config.js' },
-  { template: 'WebpackServeConfig', output: 'webpack.serve.config.js' },
+  { template: 'StorybookMain', output: '.storybook/main.js' },
```
[the remark this answers is not in the document]
Comment (ecraig12345) [reply scripts/create-package.js:59]: Also why did you get rid of the webpack config?

[the remark this answers is not in the document]
Comment (dzearing) [reply scripts/create-package.js:59]: Hmm. I didn't think it was needed unless we create UMD bundles for every library. On second thought, maybe we should?

File: scripts/templates/create-package/EmptyPackageJson.mustache (hunk 2 above)
Comment (dzearing) [reply scripts/templates/create-package/EmptyPackageJson.mustache:51]: This was added by ken to detect which package versions are used in an application.

File: scripts/create-package.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (ecraig12345) [reply scripts/create-package.js:59]: We currently create UMD bundles for every package, but I don't know if we necessarily need to do that.

**Pull request merged**